cookiecutter/exceptions.py

```diff
@@ -113,3 +113,26 @@ class RepositoryNotFound(CookiecutterException):
     """
     Raised when the specified cookiecutter repository doesn't exist.
     """
+
+
+class RepositoryCloneFailed(CookiecutterException):
+    """Raised when a cookiecutter template can't be cloned."""
+
+    BRANCH_ERRORS = [
+        'error: pathspec',
+        'unknown revision',
+    ]
```

Comment:
Yup, here are the actual error messages: https://github.com/audreyr/cookiecutter/pull/778/files/55f6f0215a5293b8dc6baafb978e85bc725ffb45#diff-8327dc289ba1c55ec5be92a9f27829ffR186